services/ui-src/src/components/drawers/ReportDrawer/ReportDrawer.tsx

```diff
@@ -8,13 +8,14 @@ import { useUser } from "utils";
 // types
 import { AnyObject, CustomHtmlElement, FormJson, EntityShape } from "types";
 // constants
-import { closeText, saveAndCloseText } from "../../constants";
+import { closeText, saveAndCloseText } from "../../../constants";
 
 export const ReportDrawer = ({
   selectedEntity,
   verbiage,
   form,
   onSubmit,
+  entityType,
```

Comment:
Nit: Would list this up by `selectedEntity` to standardize with how you pass it through from `ModalDrawerReportPage`. Same note down on lines 33 & 70. And `Drawer` line 76. This might be the most nit nit of all nit so feel free to disregard.

Reply:
I left the props in required order because I think it makes a little more sense that way 🤷